### PR TITLE
Add Holesky Etherscan and faucet

### PR DESCRIPTION
**Pull request by 0xV4L3NT1N3**

### CODE REVIEW

File: _data/chains/eip155-17000.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   ],
   "faucets": [
     "https://faucet.holesky.ethpandaops.io",
-    "https://holesky-faucet.pk910.de"
+    "https://holesky-faucet.pk910.de",
+    "https://chat.blockscan.com/index?a=0x035219a183f9e31514e0d7c23a85e3e76d7bf167"
```
Comment (ligi) [inline _data/chains/eip155-17000.json:12]: this does not look like a faucet to me

Comment (0xV4L3NT1N3) [reply _data/chains/eip155-17000.json:12]: This is a chat faucet, similar to the ones on Discord 

https://x.com/etherscan/status/1707712285115056469?s=46

Comment (0xV4L3NT1N3) [reply _data/chains/eip155-17000.json:12]: Do let me know if this is not suitable, will remove it from the PR!

Comment (ligi) [reply _data/chains/eip155-17000.json:12]: bit on the fence here - when I open this page I just get to some start-page

**Comment by 0xV4L3NT1N3** [comment]
Good point, we haven't built a solid landing page with instructions as this was prepared in time for Holesky's launch.

Removing it and keeping the explorer for now!

File: _data/chains/eip155-17000.json (hunk 1 above)
Comment (nnnnnnsssss) [reply _data/chains/eip155-17000.json]: Reechek

**Review by ligi — APPROVED** [review]

**Pull request merged**